contracts/contract/dao/RocketDAOProposal.sol

```diff
@@ -132,30 +131,29 @@ contract RocketDAOProposal is RocketBase, RocketDAOProposalInterface {
         require(getTotal() >= _proposalID && _proposalID > 0, "Invalid proposal ID");
         // Get the amount of votes for and against
         uint256 votesFor = getVotesFor(_proposalID);
-        // uint256 votesAgainst = getVotesAgainst(_proposalID);
+        uint256 votesAgainst = getVotesAgainst(_proposalID);
         // Now return the state of the current proposal
         if (getCancelled(_proposalID)) {
             // Cancelled by the proposer?
             return ProposalState.Cancelled;
             // Has it been executed?
         } else if (getExecuted(_proposalID)) {
             return ProposalState.Executed;
-            // Has it expired?
-        } else if (block.number >= getExpires(_proposalID)) {
-            return ProposalState.Expired;
-            // Vote was successful, is now awaiting execution
-        } else if (votesFor >= getVotesRequired(_proposalID)) {
-            return ProposalState.Succeeded;
             // Is the proposal pending? Eg. waiting to be voted on
         } else if (block.number <= getStart(_proposalID)) {
```

Comment:
Not related to this PR but shouldn't this be < instead of <=?

If voting starts at block 1000 and ends at block 1100, I would expect to be able to vote in block 1000 and no longer be able to vote in block 1100.

Reply:
Yeah that makes sense, will add it :+1: